javascripts/chooseplayer.js

```diff
@@ -9,16 +11,22 @@ var FoodFight = (function(choose){
     FoodFight.setUpGame(currentHero, currentEnemy);
   });
 
+  heroBtn.click(function() {
+    selected = heroBtn.filter(":checked").val();
+    console.log(selected);
+  });
+
   choose.setHero = function(){
-    var hero = FoodFight.Larry;
+    var name = userName.val()
+    var hero = new FoodFight[selected](name);
```

Comment:
the name on line 21 is needed because it's passing in the name variable from line 20. It's bringing back the full hero object and passing the user's name into it